tests/training/multi_gpu_test.py

```diff
@@ -4,7 +4,7 @@
 import keras.backend as K
```

Comment:
You should probably rename this test.  And can you add tests for the methods you added to `train_utils`?